File: derash/lib/models/station.dart
class Station {
  String? id;
  String stationemail;
  String stationlocation;
  Station({
    this.id,
    required this.stationemail,
    required this.stationlocation,
  });

  factory Station.fromJson(Map<String, dynamic> json){
    return Station(
        id: json["_id"],
        stationemail: json["stationemail"],
        stationlocation: json["stationlocation"],
      );
  }
    Map<String, dynamic> toJson() => {
         "id":id,
        "stationemail": stationemail,
        "stationlocation": stationlocation,
      };
}

File: derash-api/models/Station.js
const mongoose = require("mongoose");
const StationSchema = new mongoose.Schema({
        stationame: {
            type: String,

        },
        stationemail: {
            type: String,
            required: true,
            unique: true
        },
        stationlocation: {
            type: String,
            required: true,
        },
    }, {
        timestamps: true
    },

);

module.exports = mongoose.model("Station", StationSchema);